setup.cfg

```
@@ -37,23 +38,30 @@ classifiers =
 
 
 [options]
-python_requires = >=2.6, !=3.0, !=3.1, !=3.2, !=3.3, !=3.4
 packages = find:
 install_requires =
     numpy
+python_requires = >=2.6, !=3.0, !=3.1, !=3.2, !=3.3, !=3.4
```

```suggestion
python_requires = >=2.6, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
```

This doesn't work as written, you picked it up from an incorrect example I wrote, I think.